lib/socketlabs/injectionapi/core/send_validator.rb

```diff
@@ -44,7 +45,7 @@ def validate_credentials(server_id, api_key)
             SendResponse.new(result=SendResult.enum["AuthenticationValidationFailed"])
           end
 
-          if server_id.nil? || server_id.empty?
+          if server_id.nil? || (!server_id.is_a?(Integer) && server_id.empty?)
```

Comment:
Checks if `server_id` is not an integer before calling `.empty?`

Integers don't have this helper, so calling `10000.empty?` as in the example provided in the README didn't work.